Library/PushNotificationdialogTests.swift

```diff
@@ -0,0 +1,85 @@
+import XCTest
+@testable import Library
+@testable import KsApi
+
+class PushNotificationdialogTests: XCTestCase {
```

Comment:
Tiny comment: should be `PushNotificationDialogTests`?

Reply:
Absolutely! :)